content/agent/basic_agent_usage/windows.md

```diff
@@ -29,7 +29,7 @@ The execution of the Agent is controlled by a Windows service.
 There are a few major changes compare to older Datadog Windows Agent version:
 
 * the main executable name is now `agent.exe` (it was `ddagent.exe` previously)
-* Commands should be run with the command line `c:\program files\datadog\datadog-agent\embedded\agent.exe <command>`
+* Commands should be run with the command line `c:\program files\datadog\datadog agent\embedded\agent.exe <command>`
```

Comment:
You did a s/c:/C:/ below but not here. I don't have a strong preference but let's be consistent.